Update docs to include use of the new fabric gateway connector (#214)
hyperledger/caliper-benchmarks: networks/fabric/README.md

npm install --only=prod @hyperledger/caliper-cli
```

- bind to the fabric

> To bind with fabric 2.2 or higher using the legacy node-sdk 2.2 run:

Reviewer: ```suggestion
To bind with fabric 2.2 (which uses the legacy node-sdk 2.2) which can be used with a Fabric 2.2 or higher SUT run:
```

```bash
npx caliper bind --caliper-bind-sut fabric:2.2
```

> To bind with fabric 2.4 or higher and utilise the new peer-gateway service run:

Reviewer: ```suggestion
To bind with fabric 2.4 and utilise the new peer-gateway service introduced in fabric 2.4 run:
```

```bash
npx caliper bind --caliper-bind-sut fabric:2.4
```

> If you change your mind or decide to chenge the connector to use make sure to unbind first (`caliper unbind --caliper-bind-sut fabric:2.2`) before binding to the new one.

Reviewer: ```suggestion
If you wish to change the version of binding make sure to unbind your current binding (for example if you bound to fabric:2.2 unbind first with `caliper unbind --caliper-bind-sut fabric:2.2`) before binding to the new one.
```
